tests/should-pass/iati-activities/iati-activity/result/document-link/category/element-present-with-namespace-attribute.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity xmlns:example-namespace="http://example.org/example-namespace/ns#"><!-- Include namespace declaration. -->
+    <iati-identifier></iati-identifier>
+    <reporting-org type="xx" ref="xx">
+      <narrative>Organisation name</narrative>
+    </reporting-org>
+    <title>
+      <narrative>Xxxxxxx</narrative>
+    </title>
+    <description>
+      <narrative>Xxxxxxx</narrative>
+    </description>
+    <participating-org role="xx"></participating-org>
+    <activity-status code="xx"/>
+    <activity-date type="xx" iso-date="2013-11-27"/>
+    <activity-date type="xx" iso-date="2013-11-27">
+      <narrative>Some stuff ere</narrative>
+    </activity-date>
+    <result type="xx">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <document-link url="xx" format="xx">
+        <title>
+          <narrative>Sample text</narrative>
+        </title>
+        <category code="xx" example-namespace:namespace-attribute="xx"/><!-- PASS: Element added with required child element, and namespace attribute. -->
```

Comment:
here

Reply:
what's here? Guessing the unnecessary comma?